requirements.txt

```diff
@@ -0,0 +1,2 @@
+requests==2.12.4
```

Comment:
I'd just opt for `pip install -r requirements.txt` locally? Maybe I'm missing the context of your question tho